lib/admin/postgres_check.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module DatabaseHealthChecker
+  def self.postgres_up
+    user_account_exists && get_and_put_operations && active_db_connection
+  end
+
+  def self.user_account_exists
+    # Test 1: Access Records
+    UserAccount.exists?.tap do |exists|
+      Rails.logger.info('POSTGRES TEST: UserAccount records exist.') if exists
+      Rails.logger.info('POSTGRES TEST: No UserAccount records found but database is up.') unless exists
+    end
+    true
+  rescue => e
+    Rails.logger.error(
```

Comment:
Do we want to log this error on the updates? I would think we would just want to log this?

Reply:
@RachalCassity you're asking whether the `Rails.logger.info` messages above are necessary? I kind of went back and forth–I think either way would be fine. I can remove the `info` messages if you want.

Reply:
Fixed.